### PR TITLE
Feature/basic styling

### PR DESCRIPTION
**Pull request by CharelMarais**
Initiated basic design concepts

### CODE REVIEW

File: src/app/components/firestore-add-item/firestore-add-item.component.html [hunk 1]
```diff
@@ -1,38 +1,175 @@
 <p>firestore-add-item works!</p>
```
Comment (MicNow27) [inline src/app/components/firestore-add-item/firestore-add-item.component.html:1]: Can remove this

File: src/app/components/firestore-add-item/firestore-add-item.component.ts [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+  test(tag: any) {
+    console.log(this.tagValue);
+  }
+
+  log(): void {
+    console.log('click dropdown button');
+  }
 }
+
+// firebaseStore.addItineraryItem(
```
Comment (MicNow27) [inline src/app/components/firestore-add-item/firestore-add-item.component.ts:34]: Remember to remove commented-out code

**Review by MicNow27 — COMMENTED** [review]
Nice :)

**Comment by CharelMarais** [comment]
I will make the adjustments in the advanced styling branch

**Pull request merged**